target/linux/brcm47xx/image/Makefile

```diff
@@ -815,23 +917,29 @@ endef
 TARGET_DEVICES += linksys-e4200-v1
 
 define Device/netgear-wgr614-v10-na
-  DEVICE_TITLE := Netgear WGR614 v10 North America
+  DEVICE_VENDOR := Netgear
+  DEVICE_MODEL := WGR614
+  DEVICE_VARIANT := v10 North America
```

Comment:
One might want to abbreviate that (although that's not a "mistake" of the author...)